README.md

```diff
@@ -0,0 +1,48 @@
+# conference
+Project: HTML/CSS & JavaScript capstone project - Conference page.
+ 
+This Project is based on an online website for a conference. 
+A responsive design
+ 
+![screenshot](https://github.com/aliabbani/Conference-page/blob/mobile-first/images/readme.png)
+ 
+ 
+## Built With
+- HTML
+- CSS
+- JavaScript
+- Coffee
+ 
+## Live Demo
+[Live Demo Link](https://aliabbani.github.io/Conference-page/)
+ 
+
+## Authors
+
```

Comment:
- [ ] Kindly include getting started instructions for non-technical people to easily run the project.

Reply:
Alright, thank you

Reply:
This issue is well resolved! ✔️